src/views/text-parsing/ui/text-parsing-options-setting.tsx

```diff
@@ -0,0 +1,205 @@
+'use client'
+
+import { Button, Fieldset, Group, InputLabel, TextInput } from '@mantine/core'
+import { isNotEmpty, useForm } from '@mantine/form'
+import { useEffect } from 'react'
+import toast from 'react-hot-toast'
+import { useOptionListSuspenseQuery } from '@/entities/attendance-option'
+import type { DayKey, ParsingOptions } from '@/entities/text-parsing'
+import {
+  DAY_OPTIONS_LABEL,
+  modifyParsingOptions,
+  modifyTextParsingOptionsWithRevalidate,
+  useTextParsingStore,
+} from '@/entities/text-parsing'
+import { Exception } from '@/shared/api'
+import { cn } from '@/shared/lib'
+import { AdminContainer, AdminTitle, Icon } from '@/shared/ui'
+import { parseTextSafely } from '../lib'
+
+export interface TextParsingOptionsSettingProps {
+  parsingOptions: ParsingOptions
+}
+
+export const TextParsingOptionsSetting: React.FC<TextParsingOptionsSettingProps> = ({
+  parsingOptions,
+}) => {
+  const { data: attendanceOptions } = useOptionListSuspenseQuery()
+
+  const text = useTextParsingStore((state) => state.text)
+  const { setOptions } = useTextParsingStore((state) => state.actions)
+
+  const form = useForm({
+    mode: 'uncontrolled',
+    initialValues: parsingOptions,
+    validate: {
+      delimiter: {
+        person: isNotEmpty(),
+        line: (value) => (JSON.stringify(value).replaceAll('"', '') === '' ? true : false),
```

Comment:
[질문]
해당 validate 함수에 걸러진 경우 예외 메시지가 올바르게 표시되는지 여쭤봐도 될까요?

[제안]
아래의 코드와 같이 작성하면 더욱 깔끔하고 간결하게 관리할 수 있을 듯 한데 어떠신가요? 삼항연산자에서 boolean 값을 비교식과 동일하게 내보내는 것이 조금 어색하게 느껴지네요.

<br />

```suggestion
        line: (value) => !JSON.stringify(value).replaceAll('"', ''),
```

Reply:
![image](https://github.com/user-attachments/assets/6a567f08-4936-44b8-86d6-b5e6b77afae5)

제안해주신 사항은 반영하였고, 질문하신 부분은 값을 입력하지 않고 저장 버튼을 클릭했을 때 위 이미지처럼 border가 빨간색으로 변경됩니다. 예외 메시지를 표시할 경우, fieldset에서 input의 기본 error 스타일링이 라벨과 정렬을 맞추는 것이 쉽지 않아 모든 필드에서 예외 메시지는 띄우지 않고 border의 색상 변경만 적용했습니다.

그런데 아무래도 메시지를 보여주는 것이 UX 측면에서 좋을 것 같아서 조금 고민해봤는데, 옵션 중 하나라도 입력되지 않은 경우 아래 이미지처럼 에러 메시지를 보여주는 것은 어떨지 여쭤보고 싶습니다!

![image](https://github.com/user-attachments/assets/190c6d3f-3d3f-4189-8f37-194c352733e8)

Reply:
좋습니다! 현재 validate 로직이 인풋의 빈값만 체크하는 경우라서 괜찮을 것 같네요.